markets/bfp-market/test/integration/modules/PerpAccountModule.mergeAccounts.test.ts

```diff
@@ -47,23 +44,40 @@ describe('PerpAccountModule mergeAccounts', () => {
       signer: fromTrader.signer,
       accountId: toTraderAccountId,
     };
-    await BfpMarketProxy.connect(fromTrader.signer)['createAccount(uint128)'](toTraderAccountId);
+
+    await withExplicitEvmMine(
```

Comment:
Latest Cannon version may have borked the reliability of this so a `withExplicitEvmMine` was added to prevent timeouts.